modules/administration/pages/auth-methods.adoc

```diff
@@ -2,99 +2,102 @@
 = Authentication Methods
 
 
+
 == Authentication Via PAM
 
 
-As security measures become increasingly complex, SUSE Manager supports network-based authentication systems via Pluggable Authentication Modules (PAM). PAM is a suite of libraries that allows to integrate SUSE Manager with a centralized authentication mechanism, thus eliminating the need to remember multiple passwords.
-SUSE Manager supports LDAP, Kerberos, and other network-based authentication systems via PAM.
-To enable SUSE Manager to use PAM in your organization's authentication infrastructure, follow the steps below.
+As security measures become increasingly complex, {productname} supports network-based authentication systems via Pluggable Authentication Modules (PAM).
```

Comment:
I think the first part of this sentence is needless, it's kinda just making a general assumption about the world. It's more authoritative to just launch straight in:

```suggestion
{productname} supports network-based authentication systems via Pluggable Authentication Modules (PAM).
```